engines/bastion_katello/app/assets/javascripts/bastion_katello/packages/details/package-details-repositories.controller.js

```diff
@@ -22,7 +22,7 @@ angular.module('Bastion.packages').controller('PackageDetailsRepositoriesControl
             'organization_id': CurrentOrganization
         };
 
-        repositoriesNutupane = new Nutupane(Repository, params, 'repositories');
+        repositoriesNutupane = new Nutupane(Repository, params);
```

Comment:
This change fixed ^ error. I noticed this when I based OSTree Branches search on Packages and had to fix in the other place.